packages/components/src/components/card/card.tsx

```diff
@@ -30,8 +30,19 @@ export class card {
   render() {
     return (
       <div class={this.getCssClassMap()}>
+        <div class="card__header">
+        <h4 class="card-title">Testtitel Header</h4>
+        <h5 class="card-subtitle">Subtitel Header</h5>
+          <slot name="header" />
+        </div> 
+        <img class="card-img-top" src="http://placehold.it/400x300" alt="Card image"></img>
```

Comment:
`class="card-img-top"` should be `class="card__img-top"`

Reply:
also at `card-title ` and `card-subtitle` ?

Reply:
yes please